ui/ts/pages/cluster.ts

```diff
@@ -206,6 +235,8 @@ module AdminViews {
             } else {
               let allStats: Models.Proto.NodeStatus[] = nodeStatuses.allStatuses();
               let totalStats: Models.Proto.Status = nodeStatuses.totalStatus();
+
+              axis.title = axis.titleFn(allStats);
```

Comment:
In any case, copied to RenderGraphs (even though no visualizations are currently rendered that way).